### PR TITLE
Fix server_test

### PR DESCRIPTION
**Pull request by jasonsliu**

### CODE REVIEW

File: src/not_found_handler.cc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "not_found_handler.h"
 
-RequestHandler::Status NotFoundHandler::Init(const std::string& /*uri_prefix*/,
-                                                const NginxConfig& /*config*/) {
+RequestHandler::Status NotFoundHandler::Init(const std::string& uri_prefix,
+                                                const NginxConfig& config) {
```
Comment (ryanvoong) [inline src/not_found_handler.cc:4]: the indentation is wonky here

File: src/server.cc [hunk 2]
```diff
@@ -101,6 +101,18 @@ bool Server::getServerInfo(const char* file_name, ServerInfo* info) {
       handler->Init(name, *handler_config);
       info->uri_prefix_to_handler[name] = handler;
 
+    } else if (key == "default") {
+      if(config.statements_[i]->tokens_.size() != 2
```
Comment (ryanvoong) [inline src/server.cc]: can you make sure to leave a space between all `if`s/`for`s and the open parenthesis

File: src/server.cc [hunk 3]
```diff
@@ -101,6 +101,18 @@ bool Server::getServerInfo(const char* file_name, ServerInfo* info) {
       handler->Init(name, *handler_config);
       info->uri_prefix_to_handler[name] = handler;
 
+    } else if (key == "default") {
```
Comment (ryanvoong) [inline src/server.cc:104]: what is the "default" case for?

File: src/server_test.cc [hunk 4]
```diff
@@ -10,33 +10,27 @@
...
 TEST(ServerInfoTest, GetServerInfo) {
-	// TODO: Please fix this test. 
-
-	// ServerInfo info;
-	// EXPECT_TRUE(Server::getServerInfo("test_config", &info));
-	// EXPECT_EQ(info.port, 1890);
+	ServerInfo info;
+	EXPECT_TRUE(Server::getServerInfo("test_config", &info));
+	EXPECT_EQ(info.port, 2020);
 	// TODO: REFACTORING should change these values.
```
Comment (ryanvoong) [inline src/server_test.cc]: is this comment stale now

**Review by ryanvoong — APPROVED** [review]

File: src/not_found_handler.cc (hunk 1 above)
Comment (jasonsliu) [reply src/not_found_handler.cc:4]: This is your code...

File: src/server.cc (hunk 3 above)
Comment (jasonsliu) [reply src/server.cc:104]: It's the default handler, which is a valid keyword in the config.

File: src/not_found_handler.cc (hunk 1 above)
Comment (ryanvoong) [reply src/not_found_handler.cc:4]: looks like i dun goofed

**Pull request merged**